### ISSUE TITLE
Review images for Alt Text

### ISSUE THREAD

**Issue by AHolder1** [CAWaterBoardDataCenter/equity-data-handbook#15]
Review images used throughout - check for caption and/or alt text. If missing or incomplete - add it in.
